extension/content.js

```diff
@@ -194,6 +208,8 @@ document.addEventListener('DOMContentLoaded', () => {
 	}
 
 	if (isRepo) {
+		const isRepoRoot = location.pathname === `/${ownerName}/${repoName}/`;
```

Comment:
OK, handling both cases now in the `isRepoRoot` test and also in the construction of the `editHref`.